docs/section1/Rule1-2.md

```diff
@@ -0,0 +1,41 @@
+
+**Function Calls:** None
+
+## Rule Logic:   
+- Create a set of the PCI values from the various outputs (it only needs to exist in at least 1 output): `pci_set = set([B_0_RMD.output.performance_cost_index, B_90_RMD.output.performance_cost_index, B_180_RMD.output.performance_cost_index, B_270_RMD.output.performance_cost_index, P_RMD.output.performance_cost_index, U_RMD.output.performance_cost_index])`
+- Create a set of the PBP values from the various outputs (it only needs to exist in at least 1 output): `pbp_set = set([B_0_RMD.output.total_proposed_building_energy_cost_including_renewable_energy, B_90_RMD.output.total_proposed_building_energy_cost_including_renewable_energy, B_180_RMD.output.total_proposed_building_energy_cost_including_renewable_energy, B_270_RMD.output.total_proposed_building_energy_cost_including_renewable_energy, P_RMD.output.total_proposed_building_energy_cost_including_renewable_energy, U_RMD.output.total_proposed_building_energy_cost_including_renewable_energy])`
+- Create a set of the BBP values from the various outputs (it only needs to exist in at least 1 output): `bbp_set = set([B_0_RMD.output.baseline_building_performance_energy_cost, B_90_RMD.output.baseline_building_performance_energy_cost, B_180_RMD.output.baseline_building_performance_energy_cost, B_270_RMD.output.baseline_building_performance_energy_cost, P_RMD.output.baseline_building_performance_energy_cost, U_RMD.output.baseline_building_performance_energy_cost])`
+- Create a set of the PBP_nre values from the various outputs (it only needs to exist in at least 1 output): `pbp_nre_set = set([B_0_RMD.output.total_proposed_building_energy_cost_excluding_renewable_energy, B_90_RMD.output.total_proposed_building_energy_cost_excluding_renewable_energy, B_180_RMD.output.total_proposed_building_energy_cost_excluding_renewable_energy, B_270_RMD.output.total_proposed_building_energy_cost_excluding_renewable_energy, P_RMD.output.total_proposed_building_energy_cost_excluding_renewable_energy, U_RMD.output.total_proposed_building_energy_cost_excluding_renewable_energy])`
+- If the length of the PCI set is not 1, raise a message and return FAIL: `if len(pci_set) != 1: outcome = FAIL and raise_message "Ruleset expects exactly one PCI value to be used in the project"`
+- If the length of the PBP set is not 1, raise a message and return FAIL: `if len(pbp_set) != 1: outcome = FAIL and raise_message "Ruleset expects exactly one PBP value to be used in the project"`
+- If the length of the BBP set is not 1, raise a message and return FAIL: `if len(bbp_set) != 1: outcome = FAIL and raise_message "Ruleset expects exactly one BBP value to be used in the project"`
+- If the length of the PBP_nre set is not 1, raise a message and return FAIL: `if len(pbp_nre_set) != 1: outcome = FAIL and raise_message "Ruleset expects exactly one PBP_nre value to be used in the project"`
```

Comment:
how are these values expected to be reported?  Usually the baseline values are averaged and then compared to the proposed.  Wouldn't it be acceptable to have 4 values for the baseline sets?  (& just one value for the proposed sets?)